UIKitRMDZ/UIKitRMDZ/Controllers/NewslineViewController.swift

```diff
@@ -0,0 +1,166 @@
+// NewslineViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Лента новостей
+final class NewslineViewController: UIViewController {
+    // MARK: - Constants
+
+    enum CountRows {
+        case stories
+        case firstPost
+        case reccomend
+        case posts
+    }
+
+    // MARK: - Private properties
+
+    private let rmLinkStorage = RMLinkStorage()
+    private let tableView = UITableView()
```

Comment:
Название лучше давать не просто tableView, а конкретнее, чтобы было понятнее что за таблица, как то так
```suggestion
    private let newslineTableView = UITableView()
```